composer.json

```diff
@@ -48,8 +48,8 @@
         "phpunit/phpunit": "^9",
         "amphp/http-server": "^3",
         "kelunik/link-header-rfc5988": "^1",
-        "psalm/phar": "~5.23",
-        "laminas/laminas-diactoros": "^2.3"
+        "psalm/phar": "*",
```

Comment:
The version will be 6 but different regarding the PHP version.

Reply:
You should only require the latest 6.8.5

Reply:
Did you read the message I wrote just above?

Reply:
Yes, and your assumption is wrong: you should always require the latest available version of Psalm.

Reply:
The composer.json of psalm/phar does not allow this.

Reply:
It does, just require psalm 6.8.5

Reply:
We should definitely not require any `*` version without an upper bound. We should likely fix the psalm version, because newer versions often make builds fail due to new issues being detected.